### DISCUSSION TITLE
Project Vic path set

### DISCUSSION THREAD

**Discussion by thiagocaparelli** [sepinf-inc/IPED#1175] [Q&A]
Hello, 
I'm using the project vic and I realized that everytime the database is updated there is a new .json created on the inbox path.
So the doubt is very simple, by default the path is set for this file: 2020-02-08 15-52 to 2022-01-06 10-37.
Do I need to do something to add the other files created?
For example, a new path for each one of the files like above:
projectVicHashSetPath = E:/Documentos/Hubstream\Inbox/2020-02-08 15-52 to 2022-01-06 10-37.json
projectVicHashSetPath = E:/Documentos/Hubstream\Inbox/2022-01-06 10-37 to 2022-03-29 14-53
projectVicHashSetPath = E:/Documentos/Hubstream\Inbox/2022-03-29 14-53 to 2022-06-10 12-51
(...)
and so on.

Thank you very much

**Comment by lfcnassif** [comment]
Unfortunately, in 3.x versions, just one single file is supported, incremental files are not, you should download all hash set from scratch as an unique file or merge those files by your own.

In the upcoming 4.0, instead of pointing to the single json path, you will be able import each of those files to the central hash database, so incremental importing will be possible.
